pandas/_testing/_warnings.py

```diff
@@ -147,9 +147,15 @@ def _assert_caught_no_extra_warnings(
 
     for actual_warning in caught_warnings:
         if _is_unexpected_warning(actual_warning, expected_warning):
-            unclosed = "unclosed transport <asyncio.sslproto._SSLProtocolTransport"
-            if actual_warning.category == ResourceWarning and unclosed in str(
-                actual_warning.message
+            # GH 44732: Don't make the CI flaky by filtering SSL-related
+            # ResourceWarning from dependencies
+            unclosed_ssl = (
+                "unclosed transport <asyncio.sslproto._SSLProtocolTransport",
+                "unclosed <ssl.SSLSocket",
+            )
+            if (
+                actual_warning.category == ResourceWarning
+                and str(actual_warning.message) in unclosed_ssl
```

Comment:
should be `any(str(actual_warning.message) in x for x in unclosed_ssl)`?

Reply:
Thanks. Ya the original check was faulty.